src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  --hover-active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.header__img_logo {
+  margin: 10px 0;
+  cursor: pointer;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  list-style: none;
+  display: flex;
+}
+
+.nav__item:nth-of-type(n + 2) {
+  margin-left: 20px;
+}
+
+.nav__item_link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__item_link:hover,
+.is-active {
+  color: var(--hover-active-color);
+}
+
+.is-active::after {
```

Comment:
Generally you want your after element to be positioned absolute according to his relative as not to take space in the block. I would personally set first `nav__item` as position relative and set `::after` on that `nav__item` with position absolute